code/code/spec/spec_objs_wind_procs.cc

```diff
@@ -0,0 +1,195 @@
+  act("Your $o <Y>glows<1> and you feel your pace quicken.", 0, ch, o, 0, TO_CHAR);
+
+  ch->affectJoin(ch, &aff, AVG_DUR_NO, AVG_EFF_YES, FALSE);
+}
+
+int doWindDamage(TBeing* ch, TBeing* vict, TObj* o) {
+  int rc = 0;
+  int dam = ::number(4, 12);
+
+  act("<W>A gust of wind bursts from $n's $o to buffet $N.<z>", FALSE, ch, o, vict, TO_NOTVICT);
+  act("<W>A gust of wind bursts from $n's $o to buffet you.<z>", FALSE, ch, o, vict, TO_VICT);
+  act("<W>A gust of wind bursts from your $o to buffet $N.<z>", FALSE, ch, o, vict, TO_CHAR);
+
+  rc = ch->reconcileDamage(vict, dam, DAMAGE_GUST);
+  if (rc == -1)
+    return DELETE_VICT;
+  return TRUE;
+}
+
+int doGust(TBeing* ch, TBeing* vict, TObj* o) {
+  int rc = 0;
+  TThing* v;
+  int dam = ::number(11, 25);
+
+  act("<W>A gust of wind bursts from $n's $o to buffet $N.<z>", FALSE, ch, o, vict, TO_NOTVICT);
+  act("The <W>wind<1> picks $N up in the <C>air<1> and slams $M to the $G.<z>", FALSE, ch, o, vict, TO_NOTVICT);
+
+  act("<W>A gust of wind bursts from $n's $o to buffet you.<z>", FALSE, ch, o, vict, TO_VICT);
+  act("The <W>wind<1> picks you up in the <C>air<1> and slams you to the $G.<z>", FALSE, ch, o, vict, TO_VICT);
+
+  act("<W>A gust of wind bursts from your $o to buffet $N.<z>", FALSE, ch, o, vict, TO_CHAR);
+  act("The <W>wind<1> picks $N up in the <C>air<1> and slams $M to the $G.<z>", FALSE, ch, o, vict, TO_CHAR);
+
+
+  // Dismount victim or dismount riders
+  if (vict->riding)
+    vict->dismount(POSITION_STANDING);
+  while ((v = vict->rider)) {
+    TBeing* tb = dynamic_cast<TBeing*>(v);
+    if (tb) {
+      rc = tb->fallOffMount(vict, POSITION_STANDING);
+      if (IS_SET_DELETE(rc, DELETE_THIS)) {
+        delete tb;
+        tb = NULL;
+      }
+    } else {
+      v->dismount(POSITION_DEAD);
+    }
+  }
+
+  vict->addToDistracted(1, FALSE);
+  vict->setPosition(POSITION_SITTING);
+  vict->addToWait(combatRound(1));
+  rc = ch->reconcileDamage(vict, dam, DAMAGE_GUST);
+  if (rc == -1)
+    return DELETE_VICT;
+  return TRUE;
+}
+
+
+// Tier 1 has wind damage only
+int gustWeaponT1(TBeing* vict, cmdTypeT cmd, const char* arg, TObj* o, TObj*) {
+  TBeing* ch;
+  ch = genericWeaponProcCheck(vict, cmd, o, 3);
+  if (!ch)
+    return FALSE;
+  
+  return doWindDamage(ch, vict, o);
+}
+
+// Tier 2 has wind damge and a chance to knock down as well as celerite buff
+int gustWeaponT2(TBeing* vict, cmdTypeT cmd, const char* arg, TObj* o, TObj*) {
+  TBeing* ch;
+  TBeing* combatant;
+
+  if (!o)
+    return FALSE;
+  if (!(ch = dynamic_cast<TBeing*>(o->equippedBy)))
+    return FALSE;  // weapon not equipped (carried or on ground)
+
+  if (cmd == CMD_GENERIC_PULSE) {
+    if ( (combatant = ch->fight()) && (!::number(0,11)) ) {
+      doCeler(ch, o);
+      return TRUE;
+    }
+    return FALSE;
+  }
+
+  if (cmd == CMD_OBJ_HIT && vict) {
+    if (!::number(0, 3)) {
+      if (!::number(0, 4))
+        return doGust(ch, vict, o);
```

Comment:
`25% * 20% = 5%` chance per hit to knock the victim down with no chance for the victim to save, right?

What will happen if two characters that get lots of attacks both dual wield weapons with this proc and attack the same mob? Could this make even high level casters completely trivial?